scripts/partition_manager/partition_manager.rst

```diff
@@ -8,9 +8,8 @@ When creating an application that requires child images (for example, a bootload
 
 See :ref:`ug_multi_image` and *Building and Configuring multiple images* in :ref:`zephyr:application` for more information about multi-image builds.
 
-By default, all |NCS| samples that require multiple images are configured to use the Partition Manager.
```

Comment:
Is this sentence wrong? Imo it's crucial to state this.

Reply:
It's wrong to specify NCS samples, as it applies to all samples.

It's also wrong to state that when a sample requires multiple images, it can be configured to NOT use the PM. (This is being stated indirectly by saying "by default").

The new sentence is more correct.